src/gridstack-engine.ts

```diff
@@ -100,7 +100,11 @@ export class GridStackEngine {
 
     let didMove = false;
     const newOpt: GridStackMoveOpts = {nested: true, pack: false};
+    let counter = 0;
     while (collide = collide || this.collide(node, area, opt.skip)) { // could collide with more than 1 item... so repeat for each
+      if (counter++ > this.nodes.length * 2) {
```

Comment:
I need to see case where this happens before taking this workaround... will revert it locally.

Reply:
This is actually fixing a broken test, so it is easy to replicate:

> Chrome Headless 131.0.0.0: gridstack grid.min/max width/height should set gs-min-w to 2. FAILED
> Error: Infinite collide check

Without this safe check, the test suite is simply going in infinite loop causing "DISCONNECTED" on headless or a stuck browser when testing on real browser. In both cases, no results are produced:

>  Disconnected reconnect failed before timeout of 2000ms (ping timeout)

Reply:
FYI, I fixed the infinite loop (had NaN for maxW) but left the check in there in case...